jerry-main/main-unix.c

```diff
@@ -107,6 +107,21 @@ assert_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< f
   }
 } /* assert_handler */
 
+/**
+ * Provide the '__gc' implementation for the engine.
```

Comment:
Just a remark: after a quick search, I haven't found the `gc` string in any of the JS tests in tests/benchmarks,debugger,jerry,jerry-test-suite. Those double underscores prefix are quite unusual, at least to me (because none of our extensions - `assert` or `print` - to the standard-mandated environment use it).

And of course we don't _have_ to follow any other engines but it is worth taking a look around for best practices.